### ISSUE TITLE
Wrong colors with recommended configuration

### ISSUE THREAD

**Issue by Popkornium18** [catppuccin/tmux#376]
### Is there an existing issue outlining your problem?

- [X] I have searched the existing issues and they do not solve my problem.

### Describe your problem.

When using the recommended configuration I'm getting the wrong colors

### Paste your configuration.

```bash
mkdir -p ~/.config/tmux/plugins/catppuccin
git clone https://github.com/catppuccin/tmux.git ~/.config/tmux/plugins/catppuccin/tmux
```

```sh
set -g mouse on
set -g default-terminal "tmux-256color"

# Disable catppuccin styling windows.
set -g @catppuccin_flavor 'macchiato'
set -g @catppuccin_window_status_style "none"
set -g @catppuccin_status_left_separator "█"
set -g @catppuccin_status_right_separator "█"

set -g @catppuccin_date_time_text "%Y-%m-%d %H:%M:%S"

run ~/.config/tmux/plugins/catppuccin/tmux/catppuccin.tmux

# Style the windows. See https://man7.org/linux/man-pages/man1/tmux.1.html#STYLES for more details.
set -gF window-status-style "bg=#{@thm_surface_1},fg=#{@thm_fg}"
set -gF window-status-current-style "bg=#{@thm_peach},fg=#{@thm_crust}"

set -g window-status-format " #T | #I "
set -g window-status-current-format " #T | #I "

set -g status-left ""
set -g  status-right "#{E:@catppuccin_status_application}"
set -ag status-right "#{E:@catppuccin_status_session}"
set -ag status-right "#{E:@catppuccin_status_user}"
set -ag status-right "#{E:@catppuccin_status_host}"
set -ag status-right "#{E:@catppuccin_status_date_time}
```

### Attach screenshots.

![image](https://github.com/user-attachments/assets/96500bf5-7cc4-458a-a978-0afded02a55a)

### What tmux version are you seeing the issue on?

tmux 3.4

**Comment by inwonakng** [comment]
I seem to have the same issue with the gray background. my relevant config:

```tmux
set -g @catppuccin_flavor "mocha"
set -g status-left ""
set -g status-right "#{E:@catppuccin_status_user}"
set -ag status-right "#{E:@catppuccin_status_directory}"
set -ag status-right "#{E:@catppuccin_status_host}"
set -ag status-right "#{E:@catppuccin_status_session}"
set -g @catppuccin_window_status_style "custom"
set -g @catppuccin_window_left_separator ""
set -g @catppuccin_window_middle_separator "█"
set -g @catppuccin_window_right_separator ""
set -g @catppuccin_window_current_background "#{@thm_mauve}"
set -g @catppuccin_status_middle_separator "█"
set -g @catppuccin_window_default_text " #W"
set -g @catppuccin_window_current_text " #W"
set -g @catppuccin_pane_border_style "fg=#{@thm_surface_0}" # Use a value compatible with the standard tmux 'pane-border-style'
set -g @catppuccin_pane_active_border_style "fg=#{@thm_peach}" # Use a value compatible with the standard tmux 'pane-border-active-style'
```

**Comment by 05ck20** [comment]
I too have a problem where my status bar is gray instead of the intended dark blue color. The only color that works correctly and is like on the thumbnail is the latte flavor.

**Comment by davidfmatheson** [comment]
I noticed this starting in 1.0.2. I pinned back to v1.0.1 for now. In tpm, this would be

```
set -g @plugin "catppuccin/tmux#v1.0.1"
```

but cloning manually like OP I would guess:
```
cd ~/.config/tmux/plugins/catppuccin/tmux
git checkout v1.0.1
```

Is there a way to control this background color, similar to how I had to use `@thm_blue` to get the old default window color back?

**Comment by kjnsn** [comment]
This is mentioned at the top of the tmux man page: https://man7.org/linux/man-pages/man1/tmux.1.html#COMMAND_PARSING_AND_EXECUTION

Like, if you want dark blue: `tmux set -g status-style 'bg=darkblue'`

Also please specify the exact expectations. I can't fix something when I don't know what the expected behaviour is. If the color is wrong, what should it be? Please refer to the palette: https://catppuccin.com/palette

**Comment by inwonakng** [comment]
@kjnsn 

I was able to dig this up from another machine that I did not update. This is what it looked like when I only imported the plugin and don't set anything manually -- I'm not sure what the default flavor was. I expected the new version's title bar to use the same color. I tried changing flavors but it didn't seem to have any effect.

<img width="1470" alt="Screenshot 2024-10-15 at 10 19 18 PM" src="https://github.com/user-attachments/assets/47f4852f-4e1d-473b-a3e9-cb9bf87bc1ed">

**Comment by kjnsn** [comment]
Thank you! According to semver a major release can contain breaking changes, and one of those changes was to the status line background.

**Comment by davidfmatheson** [comment]
This change was between v1.0.1 and v1.0.2.

```
set -g @plugin "catppuccin/tmux#v1.0.1"
```

<img width="801" alt="image" src="https://github.com/user-attachments/assets/e7be678d-820a-47e9-a651-031c35d5356b">

The background seems to be transparent.

```
set -g @plugin "catppuccin/tmux#v1.0.2"
```

<img width="807" alt="image" src="https://github.com/user-attachments/assets/8b3ce12e-b19e-4319-a39f-c7e33cb67ca2">

The background has a grey color. Adding `set -g status-style 'bg=#{@thm_crust}' to my setup does not seem to help, the grey overrides it.

**Comment by kjnsn** [comment]
Are you adding it before or after you run catppuccin? If it's before then obviously it will be overridden

**Comment by davidfmatheson** [comment]
I was more commenting on the, "major releases can contain breaking changes" comment to say this was broken between minor patch releases, but regardless, a good workaround for me was to set the background explicitly to what it was before using:

```
set -g @catppuccin_status_background "#1e1e2e"
```

To determine what it was before I loaded the v1.0.1 version and then used command:

```
tmux show-option -gqv status-style
```

**Comment by Popkornium18** [comment]
> Also please specify the exact expectations.

The expectation is, that using configuration snippets in the docs results in the same look as the screenshot above them. At least for "Config 1" that is not the case.

**Comment by kjnsn** [comment]
Fixed in #377
